ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -54,11 +61,13 @@
       final TransientQueryMetadata queryMetadata,
       final long disconnectCheckInterval,
       final ObjectMapper objectMapper,
-      final CompletableFuture<Void> connectionClosedFuture
+      final CompletableFuture<Void> connectionClosedFuture,
+      final boolean oldFormat
```

Comment:
nit: can we rename this to something more meaningful?

Reply:
suggestions?

Reply:
Gone with `v1Format`. Not sure it's any better.

Reply:
Definitely better. Thanks!